### DISCUSSION TITLE
Force install/reinstall/override option

### DISCUSSION THREAD

**Discussion by rafaelmvargas** [microsoft/winget-cli#1351] [Ideas]
It would be great if there was an option to force winget to install or reinstall a package. 

Currently, I get a exit code 109 when trying to re-install Microsoft Mouse and Keyboard Center after removing it via the control panel. I'm on Windows 11 Beta. The MSI installer states I have an existing installation of the current version, which is not true (it could be possible it exists in the registry). Forcing an override would just solve problems without having to reformat. I am also running in an elevated Windows Terminal PowerShell prompt.

**Comment by denelon** [comment]
The installer would return 109 to the Windows Package Manager as well. I did a quick search and found a [report](https://answers.microsoft.com/en-us/windows/forum/all/cannot-installuninstall-microsoft-mouse-and/6adfe560-10ad-45f6-9489-834be714e5b4) referencing a [possible solution](https://support.microsoft.com/topic/fix-problems-that-block-programs-from-being-installed-or-removed-cca7d1b6-65a9-3d98-426b-e9f927e1eb4d).

**Reply by rafaelmvargas** [reply]
Thank you. I tried that multiple times before posting. Hence the suggestion of a force install option. I think I have exhausted my options including uninstalling drivers in the Device Manager. 

**EDIT:**
I have also ran `sfc /scannow` and then `DISM.exe /Online /Cleanup-image /Restorehealth` without success.

**Reply by denelon** [reply]
@rafaelmvargas I wish I knew something else that might help fix the issue. Sadly, the Windows Package Manager wouldn't be able to fix the problem either. I don't know if there is a better support contact to get more help. It looks like there is a link to "Get Support" / "Contact Us" at the [Common Microsoft Mouse and Keyboard Center Setup Issues](https://support.microsoft.com/en-us/windows/common-microsoft-mouse-and-keyboard-center-setup-issues-e814f335-6bef-d470-3430-f102007b4a81) page.
